balboa-service-kafka/src/main/scala/com/socrata/balboa/service/kafka/KafkaConsumerCLIBase.scala

```diff
@@ -28,33 +27,31 @@ trait KafkaConsumerCLIBase[K,M] extends App {
   //////////////////////////////////////////////////////////////////////////////////////////
   /////// Running the Application
 
-  override def main(args: Array[String]): Unit = {
-    super.main(args)
-    Log.info("Creating Consumer Group")
-    val g = consumerGroup()
-    Log.info("Completed Creating Consumer Group")
-
-    // Add the shutdown hook.
-    Log.info("Adding Shutdown hook")
-    scala.sys.addShutdownHook(try {
-      g.stop()
-    } catch {
-      case e: Exception => Log.warn("Unable to stop consumer group. ", e)
-      case t: Throwable => Log.warn(s"Unknown error while stopping consumer group.", t)
-    })
-
-    Log.info(s"Starting $g...")
-    g.start()
-  }
+  super.main(args)
```

Comment:
Do you still need this? If you are not overriding main, then the super.main will be invoked naturally.